translations/messages.fr.xlf

```diff
@@ -2182,6 +2190,166 @@
+        <target>Quand des pièces sont empilées, vous pouvez inspecter la pile afin de voir quelles pièces se trouvent sous le scarabée. Pour ce faire, cliquez simplement sur une pile, y combris les piles adverses, afin de voir sa composition.&lt;br/&gt;&lt;br/&gt;Votre adversaire a un scarabée au dessus d'une de vos pièces, cliquez dessus afin de voir quelle est cette pièce.</target>
+      </trans-unit>
+      <trans-unit id="6207423912703337741" datatype="html">
+        <source>Moving the grasshopper</source>
+        <target>Déplacer la sauterelle</target>
+      </trans-unit>
+      <trans-unit id="5053619000484438171" datatype="html">
+        <source>The grasshopper jumps over one or more pieces in a straight line, and stops at the first empty space encountered. It cannot jump over empty spaces, and can only move through jumps. Because it jumps, it does not adhere to the sliding restriction.&lt;br/&gt;&lt;br/&gt;Jump over three pieces with your grasshopper!</source>
+        <target>La sauterelle saute au dessus d'une ou plusieurs pièce en ligne droite, et s'arrête à la première case vide. Elle ne peut pas sauter au dessus de cases vides, et peut uniquement se déplacer en sautant. Car elle saute, elle ne doit pas respecter la restriction de glissade.&lt;br/&gt;&lt;br/&gt;Sautez au-dessus de trois pièces avec votre sauterelle !</target>
+      </trans-unit>
+      <trans-unit id="5124714891438504376" datatype="html">
+        <source>Moving the soldier ant</source>
+        <target>Déplacer la fourmi légionnaire</target>
+     </trans-unit>
+      <trans-unit id="2644663260990844906" datatype="html">
+        <source>The soldier ant is a powerful piece: it can move anywhere in the hive as long as it respects the sliding and the one hive restrictions!&lt;br/&gt;&lt;br/&gt;Your queen bee is stuck, move your soldier ant to free up your queen!</source>
+        <target>La fourmi légionnaire est une pièce puissante : elle peut se déplacer n'importe où dans la ruche tant qu'elle respecte la règle de glissade et la règle de la ruche unique !&lt;br/&gt;&lt;br/&gt;Votre reine est bloquée, déplacez votre fourmi afin de la libérer !</target>
+      </trans-unit>
+      <trans-unit id="3777509045596617031" datatype="html">
+        <source>Moving the spider</source>
+        <target>Déplacer l'arraignée</target>
+      </trans-unit>
+      <trans-unit id="8715261720341988788" datatype="html">
+        <source>Finally, the spider can move exactly 3 spaces at a time. It is not allowed to go twice through the same space in a move. As the spider crawls around other pieces, it can only move around pieces that it is in direct contact with. To do a spider move, select a spider and click on the three spaces that compose the move.&lt;br/&gt;&lt;br/&gt;Again, your queen bee is stuck but you can free it by moving your spider, do it!</source>
```

Comment:
pieces that are*

Reply:
it is! donc pas fixed ! (pieces that it is in direct contact with)

Reply:
oui mais c'est le it que je te demande de changer fieutouille :D
J'M'EN CARRETTE D'LA COUK A BRETT POLETTE !
it can only move around pieces that *are* in direct contact with *it* :D
Je que l'autisme et moi sur mon bureau demain matin sans faute, je que l'élan de la situation, je tellement sur vous, je clair Caribou, ne pas ?

Reply:
y'a double commentaire ici ! Et c'est déjà en are !